examples/ProductDiscount/ProductDiscountSetValidFromAction.json

```diff
@@ -0,0 +1,12 @@
+#%RAML 1.0 DataType
```

Comment:
I would say this was a Copy&Paste error. This is no json :D

Reply:
Weird, I have commited this: 
`{
  "action": "setValidFromAndUntil",
  "validFrom": "2001-09-11T14:05:00.000Z"
}`
Maybe it got mixed in another branch commit?